recipes/inja/all/conanfile.py

```diff
@@ -6,7 +6,7 @@
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.50.0"
+required_conan_version = ">=2.0"
```

Comment:
we can keep the old conan version
```suggestion
required_conan_version = ">=1.50.0"
```